lib/generators/geoblacklight/templates/catalog_controller.rb

```diff
@@ -265,6 +265,12 @@ class CatalogController < ApplicationController
     config.autocomplete_path = 'suggest'
   end
 
+  # @return [Array] first value is a Blacklight::Solr::Response and the second
+  #                 is a list of documents
+  def action_documents
```

Comment:
If we do end up doing something in GeoBlacklight can we use the concern approach rather than generating this into adopters application?

Reply:
Thanks for the feedback! I'll move this fix over to Blacklight.